tests/core/test_verifying.py

```diff
@@ -156,7 +156,7 @@ def test_ecr(seeder):
         # ecr auth cred is verified to be a valid credential
         assert result.status == falcon.HTTP_202
         hby.kevers[hab.pre] = hab.kever
-        auth = Authorizer(hby, vdb, eacrdntler.rgy.reger, [LEI1])
+        auth = Authorizer(hby, vdb, eacrdntler.rgy.reger, [LEI1], {"EBA Data Submitter"})
```

Comment:
Would prefer this to use an "EBA Data Submitter" constant here and in the other areas.